src/components/pages/CurriculumInfo/tabs/UnitsTab/UnitsTab.tsx

```diff
@@ -248,269 +250,323 @@ const UnitsTab: FC<UnitsTabProps> = ({ data }) => {
+                                "calc(50% - 28px)",
+                                "calc(33% - 26px)",
+                              ]}
+                              data-testid={
+                                isHighlighted
+                                  ? "highlighted-unit-card"
+                                  : "unit-card"
+                              }
                             >
-                              {index + 1}
-                            </OutlineHeading>
-                            <Heading tag={"h3"} $font={"heading-7"}>
-                              {isHighlighted && (
-                                <VisuallyHidden>
-                                  Highlighted:&nbsp;
-                                </VisuallyHidden>
-                              )}
-                              {unit.title}
-                            </Heading>
-                          </Card>
-                        );
-                      })}
-                  </Flex>
-                </Box>
-              );
-            })}
-        </GridArea>
-      </Grid>
+                              <BrushBorders
+                                color={isHighlighted ? "black" : "white"}
+                              />
+                              <OutlineHeading
+                                tag={"div"}
+                                $font={"heading-5"}
+                                $fontSize={24}
+                                $mb={12}
+                              >
+                                {index + 1}
+                              </OutlineHeading>
+                              <Heading tag={"h3"} $font={"heading-7"}>
+                                {isHighlighted && (
+                                  <VisuallyHidden>
+                                    Highlighted:&nbsp;
+                                  </VisuallyHidden>
+                                )}
+                                {unit.title}
+                              </Heading>
+                            </Card>
+                          );
+                        })}
+                    </Flex>
+                  </Box>
+                );
+              })}
+          </GridArea>
+        </Grid>
+      </Box>
+      <Flex
+        $flexDirection={["column", "row"]}
+        $background={"mint"}
+        $mt={48}
+        $pa={48}
+        $gap={24}
+      >
+        <Flex
+          $flexDirection={["column", "row"]}
+          $alignItems={["flex-start", "flex-end"]}
+          $ma={"auto"}
+          $justifyContent={["space-evenly"]}
+          $gap={24}
+        >
+          <Flex $alignItems={"flex-start"} $flexDirection={["column", "row"]}>
+            <Icon
+              name="books"
+              size={92}
+              $background={"teachersRed"}
+              $mr={40}
+              $mb={[24, 0]}
+              $color={"black"}
+            />
+
+            <Flex
+              $width={"70%"}
```

Comment:
This 70% width is preventing the text from stretching on phone view, might need to alter it to 100% on phone view